node/resolvers/search/offer.ts

```diff
@@ -96,5 +96,12 @@ export const resolvers = {
       return filteredGiftProducts
     },
     discountHighlights: propOr([], 'DiscountHighLight'),
+    spotPrice: (offer: CommertialOffer) => {
+      const sellingPrice = offer.Price
+      const spotPrice: number | undefined = offer?.Installments.find(({NumberOfInstallments, Value}) => {
+        return (NumberOfInstallments === 1 && Value < sellingPrice)
+      })?.Value;
+      return spotPrice ? spotPrice : sellingPrice
```

Comment:
You could
```suggestion
      return spotPrice ?? sellingPrice
```
But this will return `spotPrice` if it is `0` (don't know if that is possible). So in this case you can also do

```suggestion
      return spotPrice || sellingPrice
```